test/sequential/test-net-connect-local-error.js

```diff
@@ -10,14 +10,16 @@ const expectedErrorCodes = ['ECONNREFUSED', 'EADDRINUSE'];
 const optionsIPv4 = {
   port: common.PORT,
   localPort: common.PORT + 1,
-  localAddress: common.localhostIPv4
+  localAddress: common.localhostIPv4,
+  family: 4
```

Comment:
I noticed you added the `family` option in a few tests where it was using the default value before. Is it because this PR removes/breaks the default value somehow? If that's the case, it would be worth mentioning it in the docs I think.

Reply:
I would assume it's because, being options for IPv4, this configuration is seeking to get a v4 connection. Previously it would have due to reordering to always put v4 first. Now, it has to explicitly request that it wants v4.

Reply:
> family <number>: Version of IP stack. Must be 4, 6, or 0. The value 0 indicates that both IPv4 and IPv6 addresses are allowed. Default: 0.

I don't think we should need to specify it though, when using the default value both IPv4 and IPv6 should be allowed.

Reply:
```suggestion
  host: common.localhostIPv4,
```

Reply:
Or you can force the IP address, as per my suggestion
```suggestion
  host: common.localhostIPv4,
```

Previously, `host` was defined for `optionsIPv6` and not for `optionsIPv4`. I think it makes more sense to just switch the two (have a `host` in `optionsIPv4`, and not in `optionsIPv6`). Or we could have the `family` option set in `optionsIPv4` and not `optionsIPv6`, but it needs to be consistent IMHO.

Reply:
Why not have `host` and `family` in both?

Reply:
Sure, can you make this change please?